spec/factories/framework_frameworks.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+    factory :framework_framework do
```

Comment:
@nrankin again should this be `:framework_frameworks`?

Reply:
In this case no, as we are referencing the model not the table. The model is always singular in rails (expressing one instance of the thing), the table is plural (containing many of the things). 
It makes sense, nevertheless it's something that I still get confused over at times in rails! 
Just to confuse matters, factory_girl file names are always in the plural, I guess because they generate many of the model. Anyhow, that's why we have here a file called framework_frameworks, but within that we refer to framework_framework. 
And the tests will definitely not pass if I messed this up - it's why tests keep me sane ;)